systest/testcontext/context.go

```diff
@@ -287,8 +291,16 @@ func New(t *testing.T, opts ...Opt) *Context {
 		tokens <- struct{}{}
 		t.Cleanup(func() { <-tokens })
 	}
+
+	t.Cleanup(func() {
+		if t.Failed() {
+			failOnce.Do(func() { close(failed) })
+		}
+	})
 	config, err := rest.InClusterConfig()
 	require.NoError(t, err)
+	config.Burst = 20
```

Comment:
this should be already set in https://github.com/spacemeshos/go-spacemesh/pull/6178/files#diff-ab921146878b38148b9a7082a00bd8c8645357f6fbf6b8e233fe83c93eff350dR294 if I'm not mistaken.